docs/guides/migrating-from-camunda-platform-7/conceptual-differences.md

```diff
@@ -0,0 +1,149 @@
+- Optimize, Operate, and Tasklist (each one is a Java application). You can scale those applications to increase availability if you want.
+
+![Camunda Platform 7 vs Camunda Platform 8 Deployment View](../img/camunda7-vs-camunda8-deployment-view.png)
+
+Camunda Platform 8 deployments happen within Kubernetes. There are [Helm charts available](self-managed/platform-deployment/helm-kubernetes/overview.md) if you want to run Camunda Platform 8 self-managed.
+
+Camunda Platform 8 is also available as a SaaS offering from Camunda, in this case, you only need to deploy your own process solution and Camunda operates the rest.
+
+:::note
+For local development purposes, you can [spin up Camunda Platform 8 on a developer machine using Docker or Docker Compose](self-managed/platform-deployment/docker.md). Of course, developers could also create a cluster for development purposes in the SaaS offering of Camunda.
+:::
+
+### Other process solution architectures
+
+Besides Spring Boot there are also other environments being used to build process solutions.
+
+#### Container-managed engine (Tomcat, WildFly, WebSphere & co)
+
+Camunda Platform 8 doesn't provide integration into Jakarta EE application servers like Camunda Platform 7 does. Instead, Jakarta EE applications need to manually add the Zeebe client library. The implications are comparable to what is described for Spring Boot applications in this guide.
+
+![container-managed](../img/architecture-container-managed.png)
+
+#### CDI or OSGI
+
+Due to limited adoption, there is no support for CDI or OSGI in Camunda Platform 8. A lightweight integration layer comparable to [Spring Zeebe](https://github.com/camunda-community-hub/spring-zeebe) might evolve in the feature and we are happy to support this as a community extension to the Zeebe project.
+
+#### Polyglot applications (C#, NodeJS, ...)
+
+When you run your application in for example NodeJS or C#, you exchange one remote engine (Camunda Platform 7) with another (Camunda Platform 8). As Zeebe comes with a different API, you need to adjust your source code. Zeebe does not use REST as API technology, but gRPC, and you will need to leverage a [client library](apis-tools/working-with-apis-tools.md#deploy-processes-start-process-instances-and-more-using-zeebe-client-libraries) instead.
+
+![polygot architecture](../img/architecture-polyglot.png)
+
+### Plugins
+
+[**Process engine plugins**](https://docs.camunda.org/manual/latest/user-guide/process-engine/process-engine-plugins/) are not available in Camunda Platform 8, as such plugins can massively change the behavior or even harm the stability of the engine. Some use cases might be implemented using [exporters](/self-managed/concepts/exporters.md). Note that exporters are only available for self-managed Zeebe clusters and not in Camunda Platform 8 SaaS.
+
+Migrating **Modeler Plugins** is generally possible, as the same modeler infrastructure is used.
```

Comment:
Maybe call it Desktop Modeler Plugins? Otherwise it could leave the impression, that you can migrate the plugins also to the web modeler